drivers/SmartThings/philips-hue/profiles/motion-sensor.yml

```diff
@@ -0,0 +1,16 @@
+name: motion-sensor
+components:
+- id: main
+  capabilities:
+  - id: motionSensor
+    version: 1
+  - id: battery
```

Comment:
Let's move `battery` to just above `refresh` for consistency with other profiles

Reply:
Reminder on this change

Reply:
Fixed on https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1348